libs/ui-toolkit/modals/src/lib/ui-modal/simple-modal/simple-modal.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Inject, Input } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { SimpleModalData, SocialMediaArray } from './simple-modal.data';
+
+@Component({
+  selector: 'gor-simple-modal',
+  templateUrl: './simple-modal.component.html',
+  styleUrls: ['./simple-modal.component.scss']
+})
+export class SimpleModalComponent {
+  constructor(
+    public dialogRef: MatDialogRef<SimpleModalComponent>,
+
+   /**
+   * Inject needed interface SimpleModalData.
+   */
+    @Inject(MAT_DIALOG_DATA) public data: SimpleModalData
+  ) {}
+
+   /**
+   * Indicates default height value for modal.
```

Comment:
This needs some rewriting. The input does not indicate the default height value, instead it is used to set the height of the modal. Take a look at the other components and see how default values can be included in docs using @default